src/main/java/com/epam/ta/reportportal/core/events/handler/subscriber/impl/LaunchAutoAnalysisSubscriber.java

```diff
@@ -65,6 +71,7 @@ public void handleEvent(LaunchFinishedEvent launchFinishedEvent, Project project
 		} else {
 			logIndexer.indexLaunchLogs(project.getId(), launch.getId(), analyzerConfig);
 		}
+		eventPublisher.publishEvent(new LaunchEvent(launch.getId(), AUTO_ANALYSIS_FINISHED_KEY));
```

Comment:
Can it be a separate class for this type of event?

Reply:
replaced